README.md

```diff
@@ -13,8 +13,28 @@ The compiler repo can be found [here](https://github.com/softdevteam/ykrustc).
 To get started, see
 [this section in the documentation](https://softdevteam.github.io/ykdocs/dev/getting_started.html).
 
-This repo uses xtask instead of regular cargo, so instead of running `cargo
-<target>`, you will instead need to execute `cargo xtask <target>`.
+This repo contains two separately-compiled workspaces, and thus the build
+system is a little non-standard. To act recursively on both workspaces, instead
+of running `cargo <target>`, instead execute `cargo xtask <target>`.
+
+You will also need to add the `-C tracer=<kind>` flag to `RUSTFLAGS` to tell
+the build system what kind of tracer you want to use (`hw` or `sw`, although
+only `hw` works at the moment).
+
+For example, to recursively test using hardware tracing:
+```
+$ export RUSTFLAGS="-C tracer=hw"
+$ cargo xtask test
+```
+
+## Consuming yk.
+
+This repo can be consumed as a regular dependency, despite the dual-workspace
+model.
+
+(This works because `cargo build` is already recursive via `build.rs`. We were
```

Comment:
This also seems more like an internal comment than something that is worth mentioning in a ReadMe, as it is of no consequence to the end user.